department-of-veterans-affairs/caseflow
# AMA hearings table

Resolves #8421 

### Description
This PR adds the new hearings table for AMA hearings! I've also updated the hearings routes to use external_id (vacols_id for legacy hearings, uuid for ama hearings) instead of the table id so that we can differentiate between legacy and ama hearings.

### Testing Plan
1. Go to hearing prep
1. Confirm that updating the daily docket and the hearing worksheet still works
